tmp/webhook_manifests/cluster_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: dokswebhooks-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
```

Comment:
Can we have a dedicated (cluster) role right away? I wouldn't want anyone to use `cluster-admin` for this, even when the webhook is still in the development stage.